packages/pn-commons/src/utility/dom.utility.ts

```diff
@@ -12,22 +12,12 @@ export enum WaitForElementResult {
  * // waits for myQuerySelector to be injected in the DOM.
  * waitForElem('.myQuerySelector').then(() => successCbk());
  */
-export function waitForElement(selector: string) {
-  return new Promise((resolve) => {
```

Comment:
Maybe we can restore the old version. I strongly belive that the strict mode causes the malfunction in local environment